packages/swap/src/server.ts

```diff
@@ -52,6 +56,13 @@ app.use('/v2/quote', quoteMiddleware, quoteRouterV2(quoter));
 
 app.use('/trpc', maintenanceMode, trpcExpress.createExpressMiddleware({ router: appRouter }));
```

Comment:
do we need to remove this?

Reply:
can't remove it yet until all the SDK users have upgraded to the new one, will add some comments